tests/decamelize.js

```diff
@@ -0,0 +1,15 @@
+var equal = require('assert').equal;
+var decamelize = require('../decamelize');
+
+
+test('#decamelize', function(){
+  equal(decamelize('theDecamelizeStringMethod'), 'the_decamelize_string_method');
+  equal(decamelize('webkitTransform', '-'), 'webkit-transform');
+  equal(decamelize('theDecamelizeStringMethod_', ''), 'thedecamelizestringmethod_');
+  equal(decamelize('TheDecamelizeStringMethod', '-'), 'the-decamelize-string-method');
```

Comment:
did just see that `decamelize` behaves different to `dasherize` when a string starts with an uppercase letter.

`equal(dasherize('TheDasherizeStringMethod'), '-the-dasherize-string-method');`

IMO this is okay. But I will add it to the docs if you guys agree with that.

// cc @epeli @dgolman